doc/articles/migrating-from-previous-releases.md

```diff
@@ -6,6 +6,20 @@ uid: Uno.Development.MigratingFromPreviousReleases
 
 This article details the migration steps required to migrate from one version to the next when breaking changes are being introduced.
 
+## Uno Platform 5.4
+
+Uno Platform 5.4 contains breaking changes for Uno.Extensions.
+
+### UWP Support for Uno.Extensions
+
+The [`Uno.Extensions`](https://aka.platform.uno/uno-extensions) compatibility with legacy UWP apps has been removed. If your app is UWP-based and uses Uno.Extensions, in order to migrate to Uno Platform 5.4, you can keep using [previous releases of Uno.Extensions](https://github.com/unoplatform/uno.extensions/releases), or [migrate your app to WinUI](https://platform.uno/docs/articles/updating-to-winui3.html).
+
+All the other features of Uno Platform 5.4 continue to be compatible with both UWP and WinUI apps.
+
+### Updates in Uno.Extensions.Reactive
+
+The generated code has changed, make sure to [review our docs to upgrade](xuid:Extensions.Migration.Reactive) to Uno Platform 5.4.
```

Comment:
@jeromelaban We changed the UID to make it match with other upgrading documents in the extension repo:

```suggestion
The generated code has changed. Make sure to [review our docs to upgrade](xuid:Uno.Extensions.Reactive.Upgrading) to Uno Platform 5.4.
```